src/content/posts/manage-fork-repository-locally-and-remotely/index.md

```diff
@@ -9,88 +18,103 @@ On many repositories you will not have the write access. Yet, you can still prop
 
 If you have made a mistake, you can revert a `git add` by using `git reset <file1> <file2> ...` or `git reset` to unstage them all. This will keep your local changes.
 
-Don't hesitate to use `git status` between each command to clearly see the actual state of your commit! You can also use the magic command `git log --graph --oneline --all --decorate` to see the actual state of your local and distant repositories in the nicest way!
+Don't hesitate to use `git status` between each command to clearly see the actual state of your changes. You can also use the magic command `git log --graph --oneline --all --decorate` to see the actual state of your local and distant repositories in the nicest way!
 
 Be careful, `git rm <file>` exists but it will permanently remove the file from the repository!
 
-When you have finished staging all your changes, you can commit by using the `commit` command :
-```git commit -m <your_commit_name>```
+When you have finished staging all your changes, you can commit by using the `commit` command:
+```
+git commit -m <your_commit_name>
+```
+> Note that commits are intended to be meaningful **little** piece of code, generally you should create several commits per Pull Request and **avoid** fat commits  
+> As one says "commit early and commit often"
 
 Note that this will just commit your changes on your local branch.
-When you want to send these changes to the remote repository, you shall use the `push` command :
-
-```git push origin <your_branch>```
+When you want to send these changes to the remote repository, you shall use the `push` command:
+```
+git push origin <your_branch>
+```
 
-(Git commands are also integrated into some code editors such as `vs code`, but you should know that it may sometimes do some strange things, and using the terminal commands may make it easier to understand what really happens.)
+> `git` commands are also integrated into some code editors such as `VS code`, but you should know that it may sometimes do some strange things, and using the terminal commands may make it easier to understand what really happens
 
 ## Pull requesting
 
-When you have finished coding and you want to propose your changes to the origin repository, you will need to pull request.
+When you have a first version of your code and you want to propose your changes to the origin repository, you will need to pull request.
 
-To do so, you must first push your branch to the remote repository.
-When it is done, simply go on the origin repository github page, and you will have the possibility to pull request your changes.
+To do so, you must first push your branch to the remote repository.  
+When it is done, go on the origin repository GitHub page, and you will have the possibility to Pull Request your changes.
 
-[Insert image here]
+![](create-pull-request-1.png)
+> You should see this banner on the GitHub page of your fork and the original repository, it doesn't matter
```

Comment:
> You should see this banner on the GitHub page of your fork and the original repository, it doesn't matter

I do not understand what is meant by ``

Reply:
yup indeed that's not really clear
I just meant that you can click on either one (the button from your fork's page or from the original repo's page) it will create a PR either way

Reply:
feel free to suggest something else!

Reply:
```suggestion
> You should see this banner on the GitHub page of your fork and the original repository. You can click on either one.
```